themes/bootstrap3/less/vendor/bootstrap/modals.less

```diff
@@ -33,7 +33,7 @@
     .translate(0, -25%);
     .transition-transform(~"0.3s ease-out");
   }
-  &.in .modal-dialog { .translate(0, 0) }
+  &.in .modal-dialog { .translate(0, 0); }
```

Comment:
Is this the only CSS change in 3.4.1?

Reply:
Looks that way, based on https://github.com/twbs/bootstrap/compare/v3.4.0...v3.4.1